### ISSUE TITLE
Are there tests validating cross-client compatibility?

### ISSUE THREAD

**Issue by carlosrymer** [chromakode/react-html-email#62]
I'm curious to know if these exists and can be provided upfront?

**Comment by kopax** [comment]
Yes me too. I have been trying with your tool and the first mail I have sent to my self arrive on my phone.

We could see a small line with the background color but no content, while on thunderbird it look normal.

I am looking also for a way to quickly test the rendering in most used email client. Also, how can we troubleshoot?
